esp-wifi/src/lib.rs

```diff
@@ -357,7 +371,7 @@ impl EspWifiTimerSource for TimeBase {
 /// use esp_wifi::EspWifiInitFor;
 ///
 /// let timg0 = TimerGroup::new(peripherals.TIMG0);
-/// let init = esp_wifi::initialize(
+/// let init = esp_wifi::init(
```

Comment:
I know you will file another PR which will rename it to `init` but currently it's `initialize` again